redis/tests/support/mock_cluster.rs

```diff
@@ -49,6 +49,12 @@ impl MockConnectionBehavior {
         }
     }
 
+    #[must_use]
+    pub fn register_new(id: &str, handler: Handler) -> RemoveHandler {
```

Comment:
In #104 I added add_new_mock_connection_behavior function, maybe pull is required - just make sure to replace it with the new function